pkg/ssh/ssh.go

```diff
@@ -348,9 +351,9 @@ func (l *SSHTunnelList) healthCheck(e sshTunnelEntry) error {
 func (l *SSHTunnelList) removeAndReAdd(e sshTunnelEntry) {
 	// Find the entry to replace.
 	l.tunnelsLock.Lock()
-	for i, entry := range l.entries {
+	for address, entry := range l.entries {
```

Comment:
You don't need to search it like this now that it's a map, right? `delete(l.entries, e.Address)`

Reply:
Oh, you are right.